content/metadata.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 title: "Projecting genetic associations through gene expression patterns highlights disease etiology and drug mechanisms"
-date: null  # Defaults to date generated, but can specify like '2022-10-31'.
+date: 2023-09-09  # Defaults to date generated, but can specify like '2022-10-31'.
```

Comment:
This is the publication date on Nat Comm, but not sure what's the common practice here.

Reply:
I'm not sure - this seems fine to me